dipy/viz/ui.py

```diff
@@ -656,40 +642,46 @@ def add_to_renderer(self, ren):
 
         """
         super(Panel2D, self).add_to_renderer(ren)
-        for ui_item in self.ui_list:
-            ui_item.add_to_renderer(ren)
+        for element in self._elements:
+            element.add_to_renderer(ren)
 
     def get_actors(self):
         """ Returns the panel actor.
 
         """
-        return [self.panel.actor]
+        return []
 
-    def add_element(self, element, position_type, position):
-        """ Adds an element to the panel.
+    def add_element(self, element, coords):
+        """ Adds a UI component to the panel.
 
-        The center of the rectangular panel is its bottom lower position.
+        The coordinates represent an offset from the lower left corner of the
+        panel.
 
         Parameters
         ----------
         element : UI
             The UI item to be added.
-        position_type: string
-            'absolute' or 'relative'
-        position : (float, float)
-            Absolute for absolute and relative for relative
+        coords : (float, float) or (int, int)
+            If float, normalized coordinates are assumed and they must be
+            between [0,1].
+            If int, pixels coordinates are assumed and it must fit within the
+            panel's size.
 
         """
-        self.ui_list.append(element)
-        if position_type == 'relative':
-            self.element_positions.append([element, position_type, position[0], position[1]])
-            element.set_center((self.lower_limits[0] + position[0] * self.size[0],
-                                self.lower_limits[1] + position[1] * self.size[1]))
-        elif position_type == 'absolute':
-            self.element_positions.append([element, position_type, position[0], position[1]])
-            element.set_center((position[0], position[1]))
-        else:
-            raise ValueError("Position can only be absolute or relative")
+        coords = np.array(coords)
+
+        if np.issubdtype(coords.dtype, np.floating):
+            if np.any(coords < 0) or np.any(coords > 1):
+                raise ValueError("Normalized coordinates must be in [0,1].")
+
+            coords = coords * self.size
+
+        #TODO: Check if coords is outside the panel.
+
```

Comment:
I think coordinates going outside the panel is fine (in the absolute case). We should just throw a warning if this happens.